common/init.sls

```diff
@@ -45,8 +45,15 @@ host-{{ hostname }}:
 {% endfor %}
 
 {% for ssh_user in common.ssh_users %}
+{{ ssh_user }}:
+  user.present:
+    - home: /home/{{ ssh_user }}
```

Comment:
This won't work on the Macs, I believe we'll need to use /Users instead of /home there. We should do this via a Jinja variable that we pull in from the map.jinja file. I think this is also causing the Travis failure.